components/Steps/Summary.jsx

```diff
@@ -37,18 +37,33 @@ const SummaryStep = ({ formData, formSteps, formPath, onFormSubmit }) => {
   if (!formSteps) return null;
   return (
     <div>
+      <div className="lbh-table-header">
+        <h2 className="govuk-fieldset__legend--l gov-weight-lighter ">
+          {`Review details for ${formData.title} ${formData.firstName} ${formData.lastName}`}
```

Comment:
This is working only for the create new person

<img width="846" alt="Screenshot 2021-02-11 at 16 26 38" src="https://user-images.githubusercontent.com/435399/107666022-56105980-6c8e-11eb-9ce6-6027d3e058c0.png">